hive_metastore_client/hive_metastore_client.py

```diff
@@ -156,6 +164,35 @@ def add_partitions_if_not_exists(
         except AlreadyExistsException:
             pass
 
+    def add_partitions_to_table(
+        self, db_name: str, table_name: str, partition_list: List[Partition]
+    ) -> None:
+        """
+        Add partitions to a table if it does not exist.
```

Comment:
```suggestion
        Add partitions to a table.
```

Reply:
I accepted your suggestion because I got your point. But, actually, this method only adds if the partition does not exist.

Reply:
Yep, but the warning that you made on the following lines (regarding the exception itself) was redundant to this part :)